packages/builders/scripts/testing/start-buggy-sendAnywhere.js

```diff
@@ -64,18 +64,30 @@ export const startSendAnywhere = async ({
     }),
   );
 
+  const agoricNamesHangs = Far('agoricNames that hangs', {
+    // ...privateArgs.agoricNames,
+    lookup: async () => {
+      trace('agoricNames.lookup being called that will never resolve');
+      // BUG: this never resolves
+      return new Promise(() => {});
```

Comment:
Can we add a comment of what exactly will hang in the test? I tracked down that `chainHub` uses `agoricNames` with `retriable`, so I assume something in the test will trigger usage of chainHub and its retriable vow to not be resolved before upgrade, but I lost track of how exactly that happens.

Also we should provide a little more details as to why this is a valid test (and equivalent to the previous one, which had the pending promise explicitly created in the vat being upgraded). Here the buggy `agoricNames` is external to the vat being upgraded, which means upgrading the vat will not reject the result promise for the `E(agoricNames).lookup()` call. However that call is simply awaited in a `retriable` async function, which means there is a local promise created and watched (the result of the async function), which is pending at vat upgrade, and which will be rejected on upgrade.

Interestingly, the lookup result promise will remain pending forever, causing a "leak" of the promise subscription (in the real world until the decider of the promise upgrades) even though the subscriber is actually no longer interested in the promise. This is a liveslots "limitation" that I should create an issue about.